lib/membrane_h264_plugin/h26x_parser.ex

```diff
@@ -407,28 +404,33 @@ defmodule Membrane.H26x.Parser do
 
       {timestamps, %{state | au_timestamp_generator: timestamp_generator}}
     else
-      case state.nalu_parser_mod do
-        Membrane.H264.NALuParser ->
-          require Membrane.H264.NALuTypes, as: NALuTypes
-          {Enum.find(au, &NALuTypes.is_vcl_nalu_type(&1.type)).timestamps, state}
-
-        Membrane.H265.NALuParser ->
-          require Membrane.H265.NALuTypes, as: NALuTypes
-          {Enum.find(au, &NALuTypes.is_vcl_nalu_type(&1.type)).timestamps, state}
-      end
+      {first_vcl_nalu(au, state).timestamps, state}
     end
   end
 
-  @spec skip_au?(AUSplitter.access_unit(), boolean(), state()) :: {boolean(), state()}
-  defp skip_au?(au, keyframe?, state) do
-    has_seen_keyframe? = keyframe? and Enum.all?(au, &(&1.status == :valid))
+  @spec forward_au?(AUSplitter.access_unit(), boolean(), state()) :: {boolean(), state()}
+  defp forward_au?(au, keyframe?, state) do
```

Comment:
shouldn't the function that ends with `?` return bool? :p

Reply:
renamed